tests/all/integration/api/validation/test_discount_codes.py

```diff
@@ -47,6 +49,44 @@ def test_quantity_min_gt_max(self):
         with self.assertRaises(UnprocessableEntity):
             DiscountCodeSchemaTicket.validate_quantity(schema, data, original_data)
 
+    def test_percent_value_gt_hundred(self):
```

Comment:
This should be in unit test

Reply:
@iamareebjamal currently there are no tests for `discount_codes` in `unit test` directory. Moreover similar tests are already implemented in this file itself.
In case you want it to be separated, what should I do now?

Reply:
Create a new file. Even if there are old tests in this file, new tests should follow correct structure